src/DotNetWorker.Core/DotNetWorker.Core.csproj

```diff
@@ -23,16 +23,18 @@
 
   <ItemGroup>
     <PackageReference Include="Azure.Core" Version="1.42.0" />
-    <PackageReference Include="Microsoft.Extensions.Hosting" Version="8.0.1" />
-    <PackageReference Include="Microsoft.Extensions.Hosting.Abstractions" Version="8.0.1" />
-    <PackageReference Include="System.Collections.Immutable" Version="8.0.0" />
-    <PackageReference Include="System.Diagnostics.DiagnosticSource" Version="8.0.1" />
   </ItemGroup>
-  
+
   <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
-    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="8.0.2" />
-    <PackageReference Include="System.Threading.Tasks.Extensions" Version="4.5.4" />
-    <PackageReference Include="Microsoft.Extensions.Options" Version="8.0.2" />
-    <PackageReference Include="System.Text.Encodings.Web" Version="8.0.0" />
+    <PackageReference Include="System.Collections.Immutable" Version="8.0.0" />
```

Comment:
System.Collections.Immutable has a 9X version. Shouldn't we use that for net9 TFM case?

Reply:
We will be - it is a transitive reference of `Microsoft.Extensions.Hosting`, which is 9.0.0 with `net9` tfm